test/distributed/test_dynamo_distributed.py

```diff
@@ -863,8 +863,7 @@ def _add(x: torch.Tensor, y: torch.Tensor) -> torch.Tensor:
             # passing args to the staticmethod (e.g. doubly passing `self`)
             # 3 is expected here for 1 forward.
             # Graph 1 should be add and imul
-            # Graphs 2 and 3 are forward hooks on device mesh, and are fine to capture.
-            self.assertEqual(cnt.frame_count, 3)
+            self.assertEqual(cnt.frame_count, 1)
```

Comment:
@voznesenskym 
Hey Michael, want to let you know that we are removing the device_mesh initialization inside of FSDP hence the changes in test_dynamo_distributed.py. Instead, we are accepting device_mesh as a new kwarg for FSDP.

Reply:
Yeah that's fine